### DISCUSSION TITLE
How to install the sensor app?

### DISCUSSION THREAD

**Discussion by droeloe1818** [netmindz/balboa_GL_ML_spa_control#80] [Q&A]
Hello!

I'm new to ESP32 and this is my first project.

Thanks for the amazing docs! It made the wiring very easy to do.

But I get stuck on the part install the app sensor. 

How do I install this?

I've googled a bit and installed Arduino IDE but I spend several hours trying how to install the software but I really can't find it.

Can please somebody please help me setting this up?

**Comment by netmindz** [comment]
Thank you

You need to install VSCode including the platformIO extension

**Reply by droeloe1818** [reply]
Thanks for your reply, do you mind making a small tutorial for this? I now downloaded Visiual studio code but I dont understand what to do next.

Maybe it can also help others :D

**Reply by netmindz** [reply]
https://platformio.org/install/ide?install=vscode

And

https://docs.platformio.org/en/latest/integration/ide/vscode.html#quick-start
